src/swim/swim.c

```diff
@@ -116,8 +118,36 @@ swim_updates_send(struct swim_context *ctx, swim_id_t id, swim_id_t to)
 out_unlock:
 	swim_ctx_unlock(ctx);
 
-	if (rc == 0)
-		rc = ctx->sc_ops->send_message(ctx, to, upds, i);
+	if (rc == 0) {
```

Comment:
a comment here would help to understand what this block is for

Reply:
This is just debug message with full context of updates.